lib/live_debugger/components/error.ex

```diff
@@ -51,4 +54,23 @@ defmodule LiveDebugger.Components.Error do
     </.error_component>
     """
   end
+
+  defp report_issue(assigns) do
```

Comment:
These two report issue components are very similar to each other. Also, you define report issue URL two times in the code. Maybe you could extract these two to `components.ex` - thanks to this we could have only one report issue component and report issue URL declared only in one place